packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -34,7 +35,11 @@ async function getValidOriginsListFromCDN(): Promise<string[]> {
         });
       })
       .catch((e) => {
-        validateOriginLogger('validOrigins fetch call to CDN failed with error: %s. Defaulting to fallback list', e);
+        if (e.name === 'TimeoutError') {
+          validateOriginLogger('validOrigins fetch call to CDN failed due to Timeout. Defaulting to fallback list', e);
```

Comment:
>Timeout [](http://example.com/codeflow?start=77&length=7)

To make it easier for humans who read the logs, can you also add the timeout that was used when calling `fetch` (e.g., the value of `ORIGIN_LIST_TIMEOUT`) to the log statement?